src/WebJobs.Script.WebHost/Controllers/InstanceController.cs

```diff
@@ -72,18 +72,6 @@ public IActionResult GetInstanceInfo()
             return Ok(_instanceManager.GetInstanceInfo());
         }
 
-        [HttpPost]
-        [Route("admin/instance/disable")]
-        [Authorize(Policy = PolicyNames.AdminAuthLevel)]
-        public async Task<IActionResult> Disable([FromServices] IScriptHostManager hostManager)
-        {
-            _logger.LogDebug("Disabling container");
-            // Mark the container disabled. We check for this on host restart
```

Comment:
Changes LGTM. Are we absolutely sure this API is not used anywhere?

Reply:
Yes. it was removed with last release.